MAUI/CheckBox/Getting-Started.md

```diff
@@ -0,0 +1,356 @@
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }      
+    }
+}   
+
+{% endhighlight %}  
+
+## Create a Simple .NET MAUI SfCheckBox
+
+The [.NET MAUI CheckBox] (SfCheckBox) control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a [`SfCheckBox`] and configure its elements.
+
+## Add namespace for referred assemblies
+
+{% tabs %}
+{% highlight xaml %}
+
+    xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons"
+
+{% endhighlight %}
+{% highlight c# %}
+
+    using Syncfusion.Maui.Buttons;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Refer SfCheckBox control with declared suffix name for Namespace
+
+{% tabs %}
+{% highlight xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                 xmlns:local="clr-namespace:GettingStarted"
+	             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons"             
+	             x:Class="GettingStarted.MainPage">
+        <ContentPage.Content>
+            <StackLayout>
+                  <syncfusion:SfCheckBox x:Name="checkBox"/>
+            </StackLayout>
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    using Syncfusion.Maui.Buttons;
+
+    namespace GettingStarted
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                StackLayout stackLayout = new StackLayout();
+                SfCheckBox checkBox = new SfCheckBox();
+                stackLayout.Children.Add(checkbox);
+                this.Content = stackLayout;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting caption
+
+The check box caption can be defined using the [`Text`] property of [`SfCheckBox`]. This caption normally describes the meaning of the check box and it displays next to check box.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfCheckBox x:Name="checkBox" IsChecked="True" Text="CheckBox"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    SfCheckBox checkBox = new SfCheckBox();
+    checkBox.IsChecked = true;
+    checkBox.Text = "CheckBox";
+    this.Content = checkBox;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/GettingStarted/checkbox.png) 
+
+## Change the check box state
+
+The three visual states of [`SfCheckBox`] are: 
+
+* Checked
+* Unchecked
+* Indeterminate
+
+![.NET MAUI CheckBox](Images/GettingStarted/tristate.png) 
+
+You can change the state of the check box using the [`IsChecked`] [`SfCheckBox`]. In checked state, a tick mark is added to the visualization of check box.
```

Comment:
You can change the state of the check box using the [`IsChecked`] [`SfCheckBox`]. In the checked state, a tick mark is added to the visualization of the check box.